build/shade/_dotnet-test.shade

```diff
@@ -37,7 +38,18 @@ default configuration = 'Debug'
         }
 
         testArgs += noParallelTestProjects.Contains(projectName) ? " -parallel none" : "";
-        Dotnet("test" + testArgs, projectFolder);
+
+        try
+        {
+            Dotnet("test" + testArgs, projectFolder);
+        }
+        catch
```

Comment:
could this accidentally swallow any exception which really should have failed the test?